app/client-v2/src/components/HomeView/CreateProjectButton.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+import { useToastService } from "@/composables/useToastService";
+import { getApiUrl } from "@/config";
+import { SPECIES, type ProjectOverview } from "@/types/projectTypes";
+import { useFetch } from "@vueuse/core";
+import { reactive, ref } from "vue";
+import { useRouter } from "vue-router";
+
+const props = defineProps<{
+  projects: ProjectOverview[];
+}>();
+
+const { showErrorToast } = useToastService();
+const router = useRouter();
+const apiUrl = getApiUrl();
+const visible = ref(false);
```

Comment:
```suggestion
const dialogVisible = ref(false);
```
Thought this was for the button visibility at first!